docs/developers/reference/api/linea-getproof.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: linea_getProof
+description: Reference content for the linea_getProof method.
+image: /img/socialCards/lineagetproof.jpg
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# `linea_getProof`
+
+Returns the account and storage values, including the Merkle proof, of the specified account.
+The supplied block parameter must be an L2 block that has been finalized on L1.
+
+:::info important
+
+Linea currently does not support the block parameter `finalized`, meaning that to get the current
+finalised block on the L1, users must retrieve the value from the `currentL2BlockNumber` variable in
+the [LineaRollup L1 contract](https://etherscan.io/address/0xd19d4b5d358258f05d7b411e21a1460d11b0876f#readProxyContract).
+:::
+
+## Parameters
+
+- `address`: A string representing the address (20 bytes) to check for balance.
+- `storageKeys`: An array of 32-byte storage keys to be proofed and included.
+- `blockParameter`: A hexadecimal block number, or one of the string tags `latest`, `earliest`, or
+      `pending`. See the [default block parameter](https://ethereum.org/en/developers/docs/apis/json-rpc/#default-block).
+
+## Returns
+
+- `accountProof`: Provides proof data for the account:
+  - `key`: The address of the account for which the proof is generated.
+  - `leafIndex`: The index of the leaf node in the Merkle Patricia Trie that corresponds to the account.
+  - `proof`: Object containing the proof data for the account:
+    - `proofRelatedNodes`: Array of nodes in the Merkle Patricia Trie used to construct the proof.
+    - `value`: The value stored at the account's leaf node in the Merkle Patricia Trie.
+- `storageProof`: An array of storage-entries as requested. Each entry is an object with these properties:
+  - `key`: The requested storage key.
+  - `leftLeafIndex`: The index of the left leaf node in the Merkle Patricia Trie that corresponds to the
+      storage slot.
+  - `leftProof`: Contains the proof data for the left leaf node, which includes the array of nodes used
+      to construct the proof and the value stored at the account's leaf node in the Merkle Patricia Trie.
+  - `rightLeafIndex`: The index of the right leaf node in the Merkle Patricia Trie that corresponds to
+      the storage slot.
+  - `rightProof`: Contains the proof data for the right leaf node, which includes the array of nodes used
+      to construct the proof and the value stored at the account's leaf node in the Merkle Patricia Trie.
```

Comment:
Same here, `stored at the account's leaf node` should be `stored at the right leaf node`?